### ISSUE TITLE
invalid SSRC when proxying Uniden SDS200

### ISSUE THREAD

**Issue by dboune** [aler9/rtsp-simple-server#1118]
## Which version are you using?

v0.20.0

## Which operating system are you using?

- [ ] Linux amd64 standard
- [ ] Linux amd64 Docker
- [ ] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [ ] Linux arm6 Docker
- [ ] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [X] macOS amd64 Docker
- [ ] Other (please describe)

## Describe the issue

Proxying a Uniden SDS200 rtsp endpoint results in the following error:

```
2022/08/23 22:43:21 INF [path scanner1] [rtsp source] ERR: invalid transport header: invalid SSRC
```

## Describe how to replicate the issue

1. Start the server with proxy to SDS200 device configured
2. read using stream client (gstreamer, vlc, etc)

## Did you attach the server logs?

yes (see sds200-rtsp-logs.zip)

Complete log with `logLevel: debug` is provided.

## Did you attach a network dump?

yes (see sds200-rtsp-logs.zip)

A pcap showing a successful direct read from the device using GStreamer as the client, as well as an unsuccessful proxied, read through rtsp-simple-server using GStreamer as the client, are included.

## Server Logs and Network Dumps

[sds200-rtsp-logs.zip](https://github.com/aler9/rtsp-simple-server/files/9411030/sds200-rtsp-logs.zip)

**Comment by aler9** [comment]
Thanks for the detailed report, this has been fixed and the fix will be included in the next release.

**Comment by aler9** [comment]
fixed in v0.20.1
